test/test_dag.py

```diff
@@ -299,11 +283,8 @@ def test_undetermined(clear_now_and_after):
 [C_4]
 depends: 'd.txt'
 output: 'e.txt'
-
-    ''')
-    dag = get_initial_dag(graph)
-    assertDAG(
-        dag, '''
+''')
+, '''
 strict digraph "" {
```

Comment:
Can the DAGs be indented as well? You might have to add `strip` the strings in the `assertDAG` function.

Reply:
It seems `strip` is causing syntax and parsing errors. I looked it up and I think `inspect.cleandoc` or `textwrap.dedent` may work instead, so I will try using experimenting with those.

Reply:
yes, `dedent` should work. I was talking about `strip()` after splitting the strings.